certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13162 @@
+                  "description": "AVAILABLE_FIELD_1",
+                  "title": "AVAILABLE_FIELD_1",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_2": {
+                  "description": "AVAILABLE_FIELD_2",
+                  "title": "AVAILABLE_FIELD_2",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_3": {
+                  "description": "AVAILABLE_FIELD_3",
+                  "title": "AVAILABLE_FIELD_3",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_4": {
+                  "description": "AVAILABLE_FIELD_4",
+                  "title": "AVAILABLE_FIELD_4",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_5": {
+                  "description": "AVAILABLE_FIELD_5",
+                  "title": "AVAILABLE_FIELD_5",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "AVAILABLE_FIELD_6": {
+                  "description": "AVAILABLE_FIELD_6",
+                  "title": "AVAILABLE_FIELD_6",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "BEGIN_OF_CONTRACT": {
+                  "description": "BEGIN_OF_CONTRACT",
+                  "format": "date-time",
+                  "title": "BEGIN_OF_CONTRACT",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "CELLULAR_NUMBER": {
+                  "description": "CELLULAR_NUMBER",
+                  "title": "CELLULAR_NUMBER",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "CHAT_LOGIN": {
+                  "description": "CHAT_LOGIN",
+                  "title": "CHAT_LOGIN",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "CIVIL_STATUS_ID": {
+                  "description": "CIVIL_STATUS_ID",
+                  "title": "CIVIL_STATUS_ID",
+                  "type": "string",
+                  "x-ms-visibility": ""
+                },
+                "COMMENT_EMPLOYEE": {
+                  "description": "COMMENT_EMPLOYEE",
+                  "properties": {
+                    "HREF": {
+                      "description": "COMMENT_EMPLOYEE_HREF",
+                      "title": "COMMENT_EMPLOYEE_HREF",
+                      "type": "string",
+                      "x-ms-visibility": ""
+                    }
+                  },
+                  "type": "object"
+                },
+                "COST_PER_HOUR": {
+                  "description": "COST_PER_HOUR",
+                  "title": "COST_PER_HOUR
```

Comment:
@ezv-compteintegration it looks like you have mistakenly added the operation "summary", "description", "operationId", and "x-ms-visibility" inside of the parameters. please move it to the post operation scope.